src/components/AdditionalInfo/AdditionalInfo.unit.spec.jsx

```diff
@@ -7,38 +7,145 @@ import AdditionalInfo from './AdditionalInfo.jsx';
 import ExpandingGroup from '../ExpandingGroup/ExpandingGroup';
 
 describe('<AdditionalInfo/>', () => {
-  let wrapper;
-  // for the sake of more tests could be written
-  // declared beforeEach and assigned to wrapper
-  beforeEach(() => {
-    wrapper = mount(<AdditionalInfo triggerText="test" />).setState({
-      open: true,
+  describe('component', () => {
+    let wrapper;
+    // for the sake of more tests could be written
+    // declared beforeEach and assigned to wrapper
+    beforeEach(() => {
+      wrapper = mount(<AdditionalInfo triggerText="test" />).setState({
+        open: true,
+      });
     });
-  });
 
-  it('should render', () => {
-    expect(wrapper.text()).to.contain('test');
-    expect(wrapper.find('h4').length).to.equal(0);
-  });
-  it('should pass aXe check', () =>
-    axeCheck(<AdditionalInfo triggerText="test" />));
-  it('should render title container as heading', () => {
-    wrapper = mount(
-      <AdditionalInfo tagName={'h4'} triggerText="test" />,
-    ).setState({
-      open: true,
+    it('should render', () => {
+      expect(wrapper.text()).to.contain('test');
+      expect(wrapper.find('h4').length).to.equal(0);
+    });
+    it('should pass aXe check', () =>
+      axeCheck(<AdditionalInfo triggerText="test" />));
+    it('should render title container as heading', () => {
+      wrapper = mount(
+        <AdditionalInfo tagName={'h4'} triggerText="test" />,
+      ).setState({
+        open: true,
+      });
+      expect(wrapper.find('h4').length).to.equal(1);
+      wrapper.unmount();
+    });
+    it('renders both children when open is true', () => {
+      const first = wrapper.find('ExpandingGroup').props();
+      expect(first.open).to.be.true;
+      return axeCheck(
+        <ExpandingGroup open={first.open}>
+          <div className="child1" />
+          <div className="child2" />
+        </ExpandingGroup>,
+      );
     });
-    expect(wrapper.find('h4').length).to.equal(1);
-    wrapper.unmount();
   });
-  it('renders both children when open is true', () => {
-    const first = wrapper.find('ExpandingGroup').props();
-    expect(first.open).to.be.true;
-    return axeCheck(
-      <ExpandingGroup open={first.open}>
-        <div className="child1" />
-        <div className="child2" />
-      </ExpandingGroup>,
-    );
+
+  describe('analytics event', () => {
+    it('should include the correct expand action', () => {
+      const handleExpandAnalyticsEvent = e => {
```

Comment:
I'd recommend only testing for one specific part of this object per test. For example, only ensure that the `e.detail.action === 'expand'` here.

Bonus points for hoisting this function out into the `describe()` callback scope, but no worries if that's not feasible.

Reply:
I hear Chris is a little busy @bkjohnson.  Mind taking a look at this when you have time?

```js
expect(
  handleAnalyticsEvent.calledWith(
    sinon.match.hasNested('detail.action', 'collapse'),
  ),
).to.be.true;
```